packages/edit-post/src/components/header/header-toolbar/index.native.js

```diff
@@ -83,6 +90,60 @@ function HeaderToolbar( {
 		return isRTL ? buttons.reverse() : buttons;
 	};
 
+	const onInsertBlock = useCallback(
+		( blockType ) => () => {
+			insertBlock( createBlock( blockType ), undefined, undefined, true, {
+				source: 'inserter_menu',
+			} );
+		},
+		[ insertBlock ]
+	);
+
+	const renderMediaButtons = (
+		<>
```

Comment:
Can we wrap these buttons within a `ToolbarGroup` so it adds the left and right border lines?

Reply:
Updated.